### DISCUSSION TITLE
Help

### DISCUSSION THREAD

**Discussion by Amt234** [Jyers/Marlin#1296] [Q&A] [answered]
Do I also flash the firmware to the LCD screen, if not, which firmware will I be flashing to the LCD screen?

**Comment by Cal-LF** [comment, marked as answer]
The firmware is only installed to the mainboard via the usb port. The LCD screen does not have to be updated unless you want custom icons.  The screen has it's own firmware called DWIN.

**Reply by Amt234** [reply]
Where do I find the DWIN firmware?

**Reply by Cal-LF** [reply]
In the Display Firmware folder of Jyers main branch.

**Reply by Amt234** [reply]
Ok! Thank you so much.

**Reply by Amt234** [reply]
Which one do I download?

**Reply by Cal-LF** [reply]
The Gotcha set is what I installed. The Original set is there if you want to reinstall back to the original icon set.

**Reply by Amt234** [reply]
Last question which ones do I install? Do I install Firmware Sets, Icon Packs, Icon Tools, and README.md?

**Reply by Amt234** [reply]
Or just one of them?

**Reply by Cal-LF** [reply]
The instructions are below the folders. That is what I followed with success.

**Reply by Amt234** [reply]
I am confused how to copy the firmware set
